1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -1,5 +1,10 @@
+import validatePassword from './index';
 
 describe('password validator', () => {
-
+    describe('output', () => {
```

Comment:
Hey @webmarkyn, I'd come back to this an kick off your tests by leading with concrete examples.

This test reads a bit too technical.

I recommend picking your first concrete example to lead this test with instead.

There are some new demos in the platform that you can check out to refine this.